packages/schematics/angular/module/index_spec.ts

```diff
@@ -118,4 +118,75 @@ describe('Module Schematic', () => {
       .toPromise();
     expect(appTree.files).toContain('/projects/bar/custom/app/foo/foo.module.ts');
   });
+
+  describe('lazy route generator', () => {
```

Comment:
Can we add a couple of tests when the module is flat?

Ie when using the `flat` option?

From the code, I don't see that this is being handled at the moment.

Reply:
Thanks for the catch! Indeed, I haven't taken care of this. I added a single test since there aren't edge cases that I could come up with.